### PR TITLE
[5.2] Fixed slug that wasn't working with utf8 characters

### PR DESCRIPTION
**Pull request by mimoo**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Doesn't our ascii method need fixing? Also, is there any reason you sent this to the master branch that doesn't get released until the end of november this year>

File: src/Illuminate/Support/Str.php [hunk 1]
```diff
@@ -356,7 +356,12 @@ public static function singular($value)
      */
     public static function slug($title, $separator = '-')
     {
-        $title = static::ascii($title);
+        // Convert to UTF8
+        if (function_exists('mb_convert_encoding')) {
```
Comment (GrahamCampbell) [inline src/Illuminate/Support/Str.php]: You don't need this check. This function ALWAYS exists because we require the mb-string function.

**Comment by mimoo** [comment]
> Doesn't our ascii method need fixing? 

I don't know, it was wrong to convert to ascii anyway as a URL can contain utf8 chars

> Also, is there any reason you sent this to the master branch that doesn't get released until the end of november this year?

I'm new to laravel, do you want me to push that on another branch?

File: src/Illuminate/Support/Str.php (hunk 1 above)
Comment (mimoo) [inline src/Illuminate/Support/Str.php]: alright fixed!

**Comment by taylorotwell** [comment]
Bug fixes would go to 5.1 branch. This would also need a more thorough description.

**Comment by mimoo** [comment]
sorry I don't have time to further dig into the issue. I thought it was a pretty straight forward and obvious patch. Good luck.

**Comment by index2015** [comment]
Excuse me, what is the time of the Session settings do not work?

**Comment by GrahamCampbell** [comment]
> Excuse me, what is the time of the Session settings do not work?

What?

**Comment by mimoo** [comment]
that was random

**Pull request closed without merging**